cmd/svcat/output/broker.go

```diff
@@ -72,26 +79,36 @@ func WriteBrokerList(w io.Writer, outputFormat string, brokers ...servicecatalog
 }
 
 // WriteBroker prints a broker in the specified output format.
-func WriteBroker(w io.Writer, outputFormat string, broker v1beta1.ClusterServiceBroker) {
+func WriteBroker(w io.Writer, outputFormat string, broker servicecatalog.Broker) {
 	switch outputFormat {
 	case FormatJSON:
 		writeJSON(w, broker)
 	case FormatYAML:
 		writeYAML(w, broker, 0)
 	case FormatTable:
-		writeBrokerListTable(w, []servicecatalog.Broker{&broker})
+		writeBrokerListTable(w, []servicecatalog.Broker{broker})
 	}
 }
 
 // WriteBrokerDetails prints details for a single broker.
 func WriteBrokerDetails(w io.Writer, broker servicecatalog.Broker) {
 	t := NewDetailsTable(w)
 
-	t.AppendBulk([][]string{
-		{"Name:", broker.GetName()},
-		{"URL:", broker.GetURL()},
-		{"Status:", getBrokerStatusFull(broker.GetStatus())},
-	})
-
+	if broker.GetNamespace() == "" {
+		t.AppendBulk([][]string{
+			{"Name:", broker.GetName()},
+			{"Scope:", getBrokerScope(broker)},
+			{"URL:", broker.GetURL()},
+			{"Status:", getBrokerStatusFull(broker.GetStatus())},
+		})
+	} else {
+		t.AppendBulk([][]string{
+			{"Name:", broker.GetName()},
+			{"Scope:", getBrokerScope(broker)},
+			{"Namespace:", broker.GetNamespace()},
```

Comment:
an unfortunate duplication. are the fields in any order?

Reply:
They are. I suppose i could build the inner array one step at a time, and only have the Namespace column surrounded by the IF

Reply:
@MHBauer how does it look now?